### PR TITLE
Pr/small fixes

### PR DESCRIPTION
**Pull request by uli42**
SSIA

### CODE REVIEW

File: nx-X11/programs/Xserver/xkb/ddxLoad.c [hunk 1]
```diff
@@ -389,7 +389,7 @@ char tmpname[PATH_MAX];
 #ifdef __sun
         if (Pclose(out) != 0)
         {
-            ErrorF("Warning: Spurious failure reported in Pclose() runnning 'xkbcomp'.\n");
+            ErrorF("Warning: Spurious failure reported in Pclose() running 'xkbcomp'.\n");
```
Comment (mjtrangoni) [inline nx-X11/programs/Xserver/xkb/ddxLoad.c:392]: ddxLoad.c: **fix** spelling.

Comment (uli42) [reply nx-X11/programs/Xserver/xkb/ddxLoad.c:392]: d'Oh... Will fix tonight

File: doc/nxagent/README.keystrokes [hunk 2]
```diff
@@ -58,21 +58,35 @@ usually you will need Mod1 and Control. Boolean in this context means
...
+replaces the default keybindings, and only one file is read, no
+merging between different configuration files is done. This also means
+that an empty or invalid configuration file deactivates all
 keybindings.
 
 If an attribute occurs more than once in a line the last one wins.
 
+Some of the actions can open a dialog window. Depending on the user's
+answer the selected action will be taken. Some actions will only show
+a dialog informing the user about the taken action. Alls these dialogs
```
Comment (sunweaver) [inline doc/nxagent/README.keystrokes]: s/Alls/All/

File: doc/nxagent/README.keystrokes [hunk 3]
```diff
@@ -98,11 +112,15 @@ viewport_scroll_left/up/right/down
...
-  Activates/deactivates deferred screen updates.
+  Activates/deactivates deferred screen updates. With a proper
+  dialog helper (see above) it will open a dialog informing the user
+  about the action taken.
 
 ignore
   Ignores the following keystroke, nothing will happen when this
-  keystroke is pressed.
+  keystroke is pressed. The keystroke will NOT be passed to the
+  clients. If you want that the clients to see the keystroke you must
```
Comment (sunweaver) [inline doc/nxagent/README.keystrokes]: ``s/If you want that the clients/If you want the clients/```

(Remote the word "that")

**Review by sunweaver — APPROVED** [review]
Great. Two minor comments about README.keystrokes. Will fix them before pushing.

**Review by sunweaver — CHANGES_REQUESTED** [review]
This branch fails to merge now. Please rebase against latest 3.6.x. If possible, fix the comments I made while on it.

**Comment by uli42** [comment]
Fixed, including the typos

**Review by sunweaver — APPROVED** [review]

**Comment by andyleejordan** [comment]
It looks like this merge brought in the deletion of `xtestext1.h`, which seems to have broken the Debian build from the 3.6.x branch for me.

**Pull request merged**